modules/transactions/src/main/java/org/apache/ignite/internal/tx/TxManager.java

```diff
@@ -182,7 +185,7 @@ CompletableFuture<Void> finish(
      */
     CompletableFuture<Void> cleanup(
             ReplicationGroupId commitPartitionId,
-            Map<TablePartitionId, String> enlistedPartitions,
+            Map<ReplicationGroupId, String> enlistedPartitions,
```

Comment:
As mentioned above, we will also need tableIds here, because of WriteIntentSwitch logic. I guess that you are going to add it in [IGNITE-24384].

Reply:
Exactly